homework.js

```diff
@@ -0,0 +1,161 @@
+//1-----------------
+let sum = [1, 2, 3, 4, 5].reduce(add,0); 
+
+function add(sum, a) {
+    return sum + a;
+}
+
+console.log(sum);
+
+
+
+//2------------------------------------------------------
+let sum = [1, 2, 3, 4, 5];
+let num1=0;
+let num2=0;
+let add=0;
+
+
+for (let x=0 ; x<sum.length ; x++){
+
+
+
+if ((sum[x]%2)==0){
+
+ num1=sum[x];
+
+
+}}
+
+for (let x=0 ; x<sum.length ; x++){
+
+if (num1 !== sum[x] && (sum[x]%2)==0){
+
+ num2=sum[x];
+
+
+}}
+
+ function myFunc(sum) {
+  return num1+num2;;
+}
+
+
+myFunc(sum);
+
+
+//3-------------------------------------------------------------
+
+
+const avg=[8,2,2,4]
+function numAvg(num){
+    var total = 0;
+    for(var i = 0;i < num.length; i++) { 
+        total+=num[i];
+    }
+    return total/num.length;
+}
+numAvg(avg);
+
+
+
+
+
+//4=------------------------------------------------------------
+
+let txt='caterpillar'
+
+txt.split("").reverse().join("")
+
+
+
+
+//5-----------------------------------------------------------
+
+
+const addDashes=['test1' ,'teat2', 'test3']
+console.log(addDashes.join("-" ));
+
+
+
+//6 ----------------------------------------------------------
+
+const countUpAndDown=[1,2,3];
+
+for (let x=0 ; x<countUpAndDown.length ; x++){
+
+console.log(countUpAndDown[x]);
+
+
+}
+
+
+function revers(countUpAndDown) {
```

Comment:
Function that will count up to a number and back down and return a string of the climb
countUpAndDown(3)